src/bp/ui-shared/src/Contents/Components/Fields/Text.tsx

```diff
@@ -1,24 +1,34 @@
+import { FormField, FormFieldType } from 'botpress/sdk'
 import React, { FC, useEffect, useState } from 'react'
 
+import { getFieldDefaultValue } from '../../utils/fields'
 import style from '../style.scss'
 import { FieldProps } from '../typings'
 
 interface TextProps extends FieldProps {
-  type: string
+  type: FormFieldType
 }
```

Comment:
Usually we try to avoid extending interfaces, not that it changes something, but we stay consistent throughout the codebase: `type TextProps = FieldProps & { type: FormFieldType }`